### ISSUE TITLE
Problems installing on ubuntu 19.10

### ISSUE THREAD

**Issue by rvanlaar** [phoronix-test-suite/phoronix-test-suite#405]
Ubuntu 19.10 doesn't have python-sklearn as a package.
The Python3-sklearn package is available:

I'm running a compare with: `phoronix-test-suite benchmark 1906219-HV-1902022SP67`

This is my output:

```
The following dependencies are needed and will be installed: 

- build-essential
- autoconf
- libsdl2-dev
- libsdl2-image-dev
- libsdl2-mixer-dev
- libsdl2-ttf-dev
- freeglut3-dev
- libsdl1.2-dev
- libsdl-gfx1.2-dev
- libsdl-net1.2-dev
- libsdl-image1.2-dev
- libsdl-ttf2.0-dev
- libsdl-mixer1.2-dev
- bison
- flex
- libopencv-dev
- opencl-headers
- ocl-icd-libopencl1
- python-scipy
- python-sklearn
- tesseract-ocr

This process may take several minutes.
E: Package 'python-sklearn' has no installation candidate
Reading package lists...
Building dependency tree...
Reading state information...
Package python-sklearn is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source


There are dependencies still missing from the system:
- Bison
- Compiler / Development Libraries
- Flex
- OpenCL
- OpenCV
- OpenGL Utility Kit / GLUT
- Python Scipy
- Python Sklearn
- SDL 1.x Development Files
- SDL 2.x Development Files

1: Ignore missing dependencies and proceed with installation.
2: Skip installing the tests with missing dependencies.
3: Re-attempt to install the missing dependencies.
4: Quit the current Phoronix Test Suite process.
Missing dependencies action: 
```

**Comment by michaellarabel** [comment]
Yes you should install python3-sklearn. This issue is already fixed in PTS Git / 9.4.0m1.

**Comment by rvanlaar** [comment]
Thank you for your apt relpy. I can confim, it works.
